js/src/components/paid-ads/asset-group/asset-group.js

```diff
@@ -79,6 +80,7 @@ export default function AssetGroup( { campaign } ) {
 			audiences: audiences.join( ',' ),
 			budget: values.amount.toString(),
 			assets_validation: isValidAssetGroup ? 'valid' : 'invalid',
+			campaign_id: isCreation ? 'new' : campaign?.id,
```

Comment:
```suggestion
			campaign_id: isCreation ? 'new' : campaign.id,
```
@ankitrox We are already checking is `campaign` is defined when initializing `isCreation`. So here we know for sure `campaign` is defined.